generators/server/templates/pom.xml.ejs

```diff
@@ -409,6 +409,48 @@
             <scope>test</scope>
         </dependency>
 <%_ } _%>
+<%_ if (databaseType === 'sql') { _%>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
```

Comment:
do we need this ? Any way, we need to check with other dependencies as there can be a duplicated here: testcontainers is used for other options

Reply:
I'll try to remove it as soon as all test pass :)